handlers/attendee.go

```diff
@@ -0,0 +1,46 @@
+package handlers
+
+import (
+    "net/http"
+	"log"
+
+	"github.com/gin-gonic/gin"
+	eventsdb "main.go/internal/database"
+	"main.go/models"
+)
+
+// create new attendee
+func CreateAttendee(c *gin.Context) {
+	setCors(c)
+    var attendee models.Attendee
+
+    // Bind JSON to the attendee struct and error handler
+    if err := c.ShouldBindJSON(&attendee); err != nil {
+        log.Printf("ERROR: %+v", err)
+		c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		return
+    }
+
+    // Create the attendee in the database
+    newAttendee, err := eventsdb.CreateAttendee(attendee)
+	if err != nil {
+		log.Printf("ERROR: %+v", err)
+		c.IndentedJSON(http.StatusInternalServerError, nil) //server error
+		return
+	}
+
+    // Return the newly created attendee
+    c.JSON(http.StatusOK, newAttendee)
```

Comment:
Tinyyy nitpicking thing, but for a CREATE, we'd typically send back a 201 (I think StatusOK is probably a 200?).